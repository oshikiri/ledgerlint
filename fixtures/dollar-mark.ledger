2020/03/26
  Expences:Household essentials  $1.23
  Assets:Cash  $-1.24
