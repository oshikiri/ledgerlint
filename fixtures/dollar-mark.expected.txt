fixtures/dollar-mark.ledger:1 imbalanced transaction, (total amount) = -0.01 $
